### ISSUE TITLE
Documentation improvement

### ISSUE THREAD

**Issue by TomKeegasi** [SergeyTsalkov/meekrodb#33]
So the documentation doesn't make it clear how to pass a WhereClause or Ass-Array with DB::delete(...).
Obviously doing **DB::delete('myTable', $someWhereClause);** fails (since argument 2 is no string).
I was about to write a patch to fix this issue (had allready finished it) when it dawned on me (also by unraveling how DB::query parameters work) that **DB:delete('myTable', '%?', $someWhereClause);** was what I was looking for. (_Some for Ass-Array_)

**TLDR:**
Add **DB:delete('myTable', '%?', $someWhereClause);**  to "new WhereClause()" or "DB:delete()" section of the documentation. :)

**Comment by SergeyTsalkov** [comment]
Good idea! I'll add a DB::update() example with WhereClause as well!

**Comment by SergeyTsalkov** [comment]
Added a few extra examples to the WhereClause section!
